### PR TITLE
rmi by image id

### LINKED ISSUE #157: rmi by image id
## Issue Description

Type: *feature request*
currently, we support delete image by name only, which provides ability of untagging when there are many images with same id. Thus we can't delete image just one step in some situations. I think provide a new feature for deleting image by image id is reasonable. 
### Describe what feature you want
```
[root@jiangnan001 ~]# sealer images
+----------------------------------+----------------------------------------------------------------------+
|             IMAGE ID             |                              IMAGE NAME                              |
+----------------------------------+----------------------------------------------------------------------+
| 8233ad647e35d6ec32d98efe45467cb5 | registry.cn-hongkong.aliyuncs.com/sealyun/kubernetes-amd64-1:v1.19.9 |
| 8233ad647e35d6ec32d98efe45467cb5 | registry.cn-hongkong.aliyuncs.com/sealyun/kubernetes-amd64:v1.19.9   |
```
sealer rmi 8233ad647e35d6ec32d98efe45467cb5 (failed, there are more than one image)
sealer rmi 8233ad647e35d6ec32d98efe45467cb5 -f (success)

### PR DESCRIPTION
**Pull request by chaunceyjiang**
### Describe what this PR does / why we need it

rmi by image id

### Does this pull request fix one issue?

Fixes #157 

### Describe how to verify it
```


```

### CODE REVIEW

File: sealer/cmd/rmi.go [hunk 1]
```diff
@@ -17,26 +17,45 @@ package cmd
...
 	Run: func(cmd *cobra.Command, args []string) {
-		if err := image.NewImageService().Delete(args[0]); err != nil {
+		if err := runRemove(args); err != nil {
 			logger.Error(err)
 			os.Exit(1)
 		}
 	},
 }
 
+func runRemove(images []string) error {
```
Comment (allencloud) [inline sealer/cmd/rmi.go:47]: never return a non-nil error. Unreasonable!

File: utils/cli/required.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+package cli
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+// RequiresMinArgs returns an error if there is not at least min args
+func RequiresMinArgs(min int) cobra.PositionalArgs {
```
Comment (allencloud) [inline utils/cli/required.go:23]: why not directly use https://github.com/spf13/cobra/blob/eb3b6397b1b5d1b0a2cd66a9afe0520f480c0a87/args.go#L60-L67 ?

**Comment by allencloud** [comment]
Could you squash all 6 commits into a single one? @chaunceyjiang

**Review by chaunceyjiang — COMMENTED** [review]
> Could you squash all 6 commits into a single one? @chaunceyjiang

ok

**Pull request closed without merging**